_articles/importing/import-data.md

```diff
@@ -19,6 +19,7 @@ You can very easily import your data from an array of different password manager
 - Dashlane (csv)
 - Enpass (csv)
 - Firefox Password Exporter (xml)
```

Comment:
Hi,
We had already updated our docs on the web vault for this.

![image](https://user-images.githubusercontent.com/1190944/42359067-e7caca88-80ad-11e8-8f3a-5cab1f4d4e2c.png)

Here we just need to change the name to **Firefox (csv)**.

Reply:
Well, sorry that I didn't notice the change on the webpage.
thanks for your patch:)